geo/rasterio/clip/wrapper.py

```
@@ -0,0 +1,157 @@
+
+
+clip_wrapper(
+    output_path=snakemake.output.path,
+    input_raster=snakemake.input.get("raster", None),
+    like_raster=snakemake.input.get("like_raster", None),
+    like_vector=snakemake.input.get("like_vector", None),
+    cog_url=snakemake.params.get("cog_url", None),
+    bounds=snakemake.params.get("bounds", None),
+    bounds_crs=snakemake.params.get("bounds_crs", None),
+    buffer=snakemake.params.get("buffer", 0),
+    extra_options=snakemake.params.get("clip_options", []),
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)